src/App.js

```diff
@@ -8,8 +8,9 @@ const App = () => (
 
      <NavBar/>
      <Routes>
-       <Route path="/" element={<ReservationPage/>}/>
-       <Route path="/apartments" element={<ReservationDetailsPage/>}/>
+        <Route path="/" element={<ReservationPage/>}/>
```

Comment:
I guess in the main page should be Apartments, as per req "On the main page, the user can see a list of motorcycles/doctors/classes/items that you selected as a theme"